_posts/2023-10-11-14.0.0-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Apache Arrow 14.0.0 Release"
+date: "2023-10-11 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 14.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 13.0.0 release, Metehan Yildirim and Oleks V. have been invited to be committers.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
+
+## R notes
+
+For more on what’s in the 14.0.0 R package, see the [R changelog][4].
+
+## Ruby and C GLib notes
```

Comment:
@kou @mrkn can you help with those?

Reply:
@kou Could you please check the following entries?

```suggestion
## Ruby and C GLib notes

### Ruby

- Add support for prepared INSERT queries ([GH-37143](https://github.com/apache/arrow/issues/37143))
- When a prepared statement is automatically closed upon exiting a block, use the same options as when the statement was prepared ([GH-37257](https://github.com/apache/arrow/issues/37257))

### C GLib

- Support more properties of `ArrowFlight::ClientOptions` ([GH-37141](https://github.com/apache/arrow/issues/37141))
- Add support for prepared INSERT queries ([GH-37143](https://github.com/apache/arrow/issues/37143))
```

Reply:
Thanks!